src/main/java/mekanism/common/integration/curios/CuriosIntegration.java

```diff
@@ -0,0 +1,51 @@
+package mekanism.common.integration.curios;
+
+import mekanism.client.render.MekanismCurioRenderer;
+import mekanism.client.render.armor.JetpackArmor;
+import mekanism.common.integration.MekanismHooks;
+import mekanism.common.registries.MekanismItems;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.fml.InterModComms;
+import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
+import net.minecraftforge.items.IItemHandler;
+import top.theillusivec4.curios.api.CuriosApi;
+import top.theillusivec4.curios.api.SlotResult;
+import top.theillusivec4.curios.api.SlotTypePreset;
+import top.theillusivec4.curios.api.client.CuriosRendererRegistry;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+public class CuriosIntegration {
+
+    public static void sendIMC() {
+        InterModComms.sendTo(MekanismHooks.CURIOS_MODID, "register_type", () -> SlotTypePreset.BODY.getMessageBuilder().build());
+    }
+
+    public static void addListeners(final IEventBus bus) {
+        bus.addListener((FMLClientSetupEvent event) -> {
+            CuriosRendererRegistry.register(MekanismItems.JETPACK.asItem(), () -> new MekanismCurioRenderer(JetpackArmor.JETPACK));
+            CuriosRendererRegistry.register(MekanismItems.ARMORED_JETPACK.asItem(), () -> new MekanismCurioRenderer(JetpackArmor.ARMORED_JETPACK));
```

Comment:
I think you should add a helper method that basically is:
```java
private static void registerRenderers(IItemProvider... providers) {
    for (IItemProvider provider : providers) {
        if (provider.asItem() instanceof ArmorItem armor && RenderProperties.get(armor) instanceof ISpecialGear gear) {
            CuriosRendererRegistry.register(armor, () -> new MekanismCurioRenderer(gear.getGearModel(armor.getSlot())));
        } else {
            Mekanism.logger.warn("Attempted to register curios renderer for a non-special gear item.");
        }
    }
}
```

and then you could just make the `addListener` method basically be:
```java
bus.addListener((FMLClientSetupEvent event) -> registerRenderers(MekanismItems.JETPACK, MekanismItems.ARMORED_JETPACK));
```